.github/workflows/docs.yml

```diff
@@ -0,0 +1,21 @@
+name: docs
+
+on: [push]
```

Comment:
```suggestion
on:
  push:
    branches:
      - master
```

Reply:
see the todo in the PR description please